PendingReleaseNotes.md

```diff
@@ -3,88 +3,28 @@
-- Ceph daemons now run under 'ceph' user and not 'root' anymore (monitor or osd store already owned by 'root' will keep running under 'root')
-- Ceph monitors have initial support for running on PVC storage. See docs on
-  [monitor settings for more detail](Documentation/ceph-cluster-crd.md#mon-settings).
-- Ceph OSDs can be created by using StorageClassDeviceSet. See docs on [Storage Class Device Sets](Documentation/ceph-cluster-crd.md#storage-class-device-sets).
-- Rook can now connect to an external cluster, for more info about external cluster [read the design](https://github.com/rook/rook/blob/master/design/ceph-external-cluster.md) as well as the documentation [Ceph External cluster](Documentation/ceph-cluster-crd.md#external-cluster)
-- Added a new property in `storageClassDeviceSets` named `portable`:
-   - If `true`, the OSDs will be allowed to move between nodes during failover. This requires a storage class that supports portability (e.g. `aws-ebs`, but not the local storage provisioner).
-   - If `false`, the OSDs will be assigned to a node permanently. Rook will configure Ceph's CRUSH map to support the portability.
-- Rook can now manage MachineDisruptionBudgets for the OSDs (only available on OpenShift). MachineDisruptionBudgets for OSDs are dynamically managed as documented in the `disruptionManagement` section of the [CephCluster CR](Documentation/ceph-cluster-crd.md##luster-settings). This can be enabled with the `manageMachineDisruptionBudgets` flag in the cluster CR.
-- Rook can now manage PodDisruptionBudgets for the following Daemons: OSD, Mon, RGW, MDS. OSD budgets are dynamically managed as documented in the [design](https://github.com/rook/rook/blob/master/design/ceph-managed-disruptionbudgets.md). This can be enabled with the `managePodBudgets` flag in the cluster CR. When this is enabled, drains on OSDs will be blocked by default and dynamically unblocked in a safe manner one failureDomain at a time. When a failure domain is draining, it will be marked as no out for a longer time than the default DOWN/OUT interval.
-- Rook now has a new config CRD `mgr` to enable ceph manager modules
-- Flexvolume plugin now supports dynamic PVC expansion.
-- The Rook-enforced minimum memory for OSD pods has been reduced from 4096M to 2048M
+
+### EdgeFS
+
 
 ### YugabyteDB
 
-- Rook now supports YugabyteDB as storage provider. YugaByteDB is a high-performance, cloud-native distributed SQL database which can tolerate disk, node, zone and region failures automatically. You can find more information about YugabyteDB [here](https://docs.yugabyte.com/latest/introduction/)
-- Newly added Rook operator for YugabyteDB lets you easily create a YugabyteDB cluster.
-- Please follow Rook YugabyteDB operator [quickstart guide](Documentation/yugabytedb.md) to create a simple YugabyteDB cluster.
+
 
 ## Breaking Changes
 
-### Ceph
+### <Storage Provider>
```

Comment:
is it intended for these placeholder `<Storage Provider>` tag to show up in the rendered markdown?  They do not show up there, e.g. https://github.com/travisn/rook/blob/reset-pending-rel-notes/PendingReleaseNotes.md, so I just wanted to be clear.

Reply:
It's really a placeholder for devs to add to. It looks fine to me (and I'd say preferred) if it doesn't show up in the rendered view.

Reply:
sounds good, I didn't know if that was the intent